F4transkript/f4transkript.download.recipe

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Description</key>
+	<string>Downloads the latest (unlicensed) version of f4transkript</string>
+	<key>Identifier</key>
+	<string>com.github.its-unibas.download.f4transkript</string>
+	<key>Input</key>
+	<dict>
+		<key>NAME</key>
+		<string>f4transkript</string>
+		<key>DOWNLOAD_URL</key>
+		<string>https://www.audiotranskription.de/en//?smd_process_download=1&amp;download_id=2544</string>
```

Comment:
It remains to be seen if this url will hold up over time as the `download_id` is very likely to change. Why not something like: `^.*href="(.*download_id=[0-9]*)".*title="f4transkript macOS".*$` ?